index.js

```diff
@@ -1,17 +1,51 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a,b){
+      return a - b
+    });
+    this.length = this.items.length;   
+  }
+  
+  get(pos) {
+    if (pos > this.items.length){
```

Comment:
nessa verificação, vc pode ter um problema caso o pos seja negativo. Vc pode fazer uma verificação usando o próprio this.items[pos]

`
if (this.items[pos]) {
return this.items[pos]
}

throw new Error('OutOfBounds');
`